File: MiPrimeraAplicacion/app/src/main/java/com/micky/miprimeraaplicacion/backend/WebServiceAPI.java
package com.micky.miprimeraaplicacion.backend;

import java.util.List;
import retrofit2.Call;
import retrofit2.http.GET;

public interface WebServiceAPI {
    @GET("albums")
    Call<List<WSAlbums>>getAlbums();
    @GET("users")
    Call<List<WSUsers>>getUsers();

    @GET("photos")
    Call<List<WSPhotos>>getPhotos();

    @GET("lugares")
    Call<List<WSLugares>>getLugares();

    @GET("personas")
    Call<List<WSLPersonas>>getPersonas();
    @GET("informaciones")
    Call<List<WSLInformaciones>>getInformaciones();
    @GET("direcciones")
    Call<List<WSLDirecciones>>getDirecciones();
    @GET("profesiones")
    Call<List<WSLProfesiones>>getProfesiones();
}

File: PrimerKotlin/app/src/main/java/com/micky/primerkotlin/backend/WebServiceAPI.kt
package com.micky.primerkotlin.backend

import io.reactivex.Observable

import retrofit2.Call
import retrofit2.http.GET

interface WebServiceAPI {
    @GET("personas.php")
    fun getPersonasObservable(): Observable<List<WSPersonas>>
    @GET("informaciones.php")
    fun getInformacionesObservable(): Observable<List<WSInformaciones>>
    @GET("direcciones.php")
    fun getDireccionesObservable(): Observable<List<WSDirecciones>>
    @GET("profesiones.php")
    fun getProfesionesObservable(): Observable<List<WSProfesiones>>
}